rest_framework/validators.py

```diff
@@ -160,10 +160,19 @@ def __call__(self, attrs, serializer):
         queryset = self.exclude_current_instance(attrs, queryset, serializer.instance)
 
         # Ignore validation if any field is None
-        checked_values = [
-            value for field, value in attrs.items() if field in self.fields
-        ]
-        if None not in checked_values and qs_exists(queryset):
+        if serializer.instance is None:
+            checked_values = [
```

Comment:
will this change have any effect on schema migrations? thanks for the patch

Reply:
Hi @auvipy I believe there's no schema change